src/bindings/python/flux/util.py

```diff
@@ -1105,7 +1105,17 @@ def validate(self, path, conf):
         subclasses may define it to implement higher level validation.
         """
 
-    def get_format_string(self, format_name):
+    def _default_env_name(self, tool_name):
+        """
+        Get tool default environment variable name (name upper case, s/-/_/g)
+        """
+        name = tool_name if tool_name else self.name
+        prefix = name.upper().replace("-", "_")
+        if self.subtable:
+            prefix += "_" + self.subtable.upper().replace("-", "_")
+        return prefix + "_FORMAT_DEFAULT"
+
+    def get_format_string(self, format_name, tool_name=None):
```

Comment:
ahh you know what, I initially didn't because I didn't like that fact that the `name` parameter in the constructor was supposed to be the "name of the tool".  But we can just think of the the `name` as the config path, then all is good.